apis/core/types_installation.go

```diff
@@ -47,6 +47,8 @@ type InstallationSpec struct {
 	// +optional
 	Context string `json:"context,omitempty"`
 
+	Verification *Verification `json:"verification,omitempty"`
```

Comment:
I would rather not have that specified in the Installation directly but have it in the Landscaper Context.
Pros:
- It can be used by mutliple installations
- It only has to be specified once
- The PKI is bound to the OCI registry in which the component is retrieved from, which is also specified in the Context.

Reply:
Good point, i will add this. Thank you